client/src/components/PropertiesList.tsx

```diff
@@ -103,13 +128,18 @@ const PropertiesListWithoutI18n: React.FC<
     <div className="PropertiesList" ref={tableRef}>
       {isTableVisible ? (
         <FilterContextProvider>
-          {useNewPortfolioMethod ? <PortfolioFilters /> : <></>}
+          {useNewPortfolioMethod ? (
+            <PortfolioFilters logPortfolioAnalytics={logPortfolioAnalytics} />
+          ) : (
+            <></>
+          )}
```

Comment:
unsure if there was a reason for the empty div but if it's extraneous: 

```suggestion
          {useNewPortfolioMethod && (
            <PortfolioFilters logPortfolioAnalytics={logPortfolioAnalytics} /> )} 
```